src/Microsoft.SqlTools.ServiceLayer/TaskServices/SqlTask.cs

```diff
@@ -96,6 +96,13 @@ public SqlTask(TaskMetadata taskMetdata, Func<SqlTask, Task<TaskResult>> testToR
             });
         }
 
+        public void Run()
+        {
+            RunAsync().ContinueWith(task =>
```

Comment:
>ContinueWith [](start = 23, length = 12)

we probably don't need the continuewith if we aren't doing anything in the callback.